src/main/java/com/sourcegraph/webhook/Dispatcher.java

```diff
@@ -56,7 +47,7 @@ public Dispatcher(Webhook hook, EventSerializer serializer) {
 
     private static String sign(String secret, String data) {
         HmacUtils hmac = new HmacUtils(HmacAlgorithms.HMAC_SHA_256, secret);
-        return hmac.hmacHex(data);
+        return "sha256=" + hmac.hmacHex(data);
```

Comment:
This is an unrelated change, yes?

Reply:
Yes, this is to conform to GitHub specifications. In the automation code, we are using `gh.ValidateSignature` for verifying HMAC signatures